{"changes": {"content": {"new": "While I was using docker to test the handsim package I found that every gazebo command I launch ends up with:\r\n\r\n```\r\nterminate called after throwing an instance of 'std::runtime_error'\r\n  what():  locale::facet::_S_create_c_locale name not valid\r\n```\r\n\r", "old": "While I was using docker to test the handsim package I found that every gazebo command I launch ends up with:\r\n\r\n```\r\nterminate called after throwing an instance of 'std::runtime_error'\r\n  what():  locale::facet::_S_create_c_locale name not valid\r\n```\r\n\r"}}, "links": {"self": {"href": "data/repositories/osrf/handsim/issues/91/changes/18111867.json"}, "html": {"href": "#!/osrf/handsim/issues/91#comment-18111867"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/handsim/issues/91.json"}}, "type": "issue", "id": 91, "repository": {"links": {"self": {"href": "data/repositories/osrf/handsim.json"}, "html": {"href": "#!/osrf/handsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{9c7d89af-6eb6-40b1-9f69-da1ae9d129ce}ts=c_plus_plus"}}, "type": "repository", "name": "handsim", "full_name": "osrf/handsim", "uuid": "{9c7d89af-6eb6-40b1-9f69-da1ae9d129ce}"}, "title": "locale runtime error when using ogre3d package"}, "created_on": "2015-05-14T18:21:59.282082+00:00", "user": {"display_name": "Jose Luis Rivero", "uuid": "{d12309b2-f745-42ee-b119-aec4fcdf81fe}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Bd12309b2-f745-42ee-b119-aec4fcdf81fe%7D"}, "html": {"href": "https://bitbucket.org/%7Bd12309b2-f745-42ee-b119-aec4fcdf81fe%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/109284c8b83411dbc7492138f6167e9ed=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsJR-5.png"}}, "nickname": "Jose Luis Rivero", "type": "user", "account_id": "557058:155a32e2-420c-4d50-98e0-0e722f63f906"}, "message": {"raw": null, "markup": "markdown", "html": "", "type": "rendered"}, "type": "issue_change", "id": 18111867}